packages/braid-design-system/src/lib/components/Tag/Tag.docs.tsx

```diff
@@ -66,68 +63,98 @@ const docs: ComponentDocs = {
-                id="clear-2"
-              >
-                Two
-              </Tag>
-            ) : null}
-            {!getState('clearThree') ? (
-              <Tag
-                onClear={() => toggleState('clearThree')}
-                clearLabel={'Clear "Three"'}
-                id="clear-3"
-              >
-                Three
-              </Tag>
-            ) : null}
-            <Text tone="secondary">
-              <TextLinkButton
-                weight="weak"
-                hitArea="large"
-                onClick={() => {
-                  setState('clearOne', false);
-                  setState('clearTwo', false);
-                  setState('clearThree', false);
-                }}
-              >
-                Reset
-              </TextLinkButton>
-            </Text>
-          </Inline>,
-        ),
+      Example: ({ setDefaultState, getState, setState }) => {
+        const { code, value } = source(
+          <>
+            {setDefaultState('selected', ['One', 'Two', 'Three'])}
+
+            <Stack space="large">
+              <Stack space="small">
+                <Text size="xsmall" tone="secondary">
+                  ADDABLE
+                </Text>
+                <Inline space="small" alignY="center">
+                  {['One', 'Two', 'Three', 'Four', 'Five']
+                    .filter((tag) => !getState('selected').includes(tag))
+                    .map((tag) => (
+                      <Tag
+                        key={tag}
+                        onAdd={() =>
+                          setState('selected', [...getState('selected'), tag])
+                        }
+                        addLabel={`Add "${tag}"`}
+                        id={`add-${tag}`}
+                      >
+                        {tag}
+                      </Tag>
+                    ))}
+                </Inline>
+              </Stack>
+              <Stack space="small">
+                <Text size="xsmall" tone="secondary">
+                  CLEARABLE
+                </Text>
+                <Inline space="small" alignY="center">
+                  {getState('selected').map((selectedTag: string) => (
+                    <Tag
+                      key={selectedTag}
+                      onClear={() => {
+                        setState(
+                          'selected',
+                          getState('selected').filter(
+                            (tag: string) => tag !== selectedTag,
+                          ),
+                        );
+                      }}
+                      clearLabel={`Clear "${selectedTag}"`}
+                      id={`clear-${selectedTag}`}
+                    >
+                      {selectedTag}
+                    </Tag>
+                  ))}
+                </Inline>
+              </Stack>
+            </Stack>
+          </>,
+        );
+
+        return {
+          code: code.replaceAll(': string', ''),
```

Comment:
Stripping type information from the above example to ensure Playroom links work as expected